framework/src/utils/PetscDMMoose.C

```diff
@@ -743,6 +743,19 @@ DMMooseGetEmbedding_Private(DM dm, IS * embedding)
               if (edof >= dofmap.first_dof() && edof < dofmap.end_dof())
                 indices.insert(edof);
           }
+          // loop over lower dimensional slave elements
+          for (const auto & bit : *(dmm->_block_ids))
```

Comment:
I am confused.. does this loop through all the elements, including the ones not on the slave?